calibrate.py

```diff
@@ -1,39 +1,55 @@
 #!/usr/bin/env python3
-# Calibration script is not yet migrated to Gen2
-raise NotImplementedError("Calibration is not yet available in Gen2 demo. To calibrate your device, switch to \"gen1_main\" branch and try again")
 
+import cv2
+import sys
+import copy
 import platform
-from contextlib import contextmanager
-
-import depthai
-from depthai_helpers.calibration_utils import *
-from depthai_helpers import utils
-import argparse
-from argparse import ArgumentParser
+import signal
+import subprocess
+import json
+import datetime
+import csv
+import time
 import numpy as np
+import os
 from pathlib import Path
 import shutil
-import json
-from depthai_helpers.version_check import check_depthai_version
-check_depthai_version()
+from datetime import datetime
+from argparse import ArgumentParser
+import argparse
+
+import depthai as dai
+from depthai_helpers.calibration_utils import *
```

Comment:
```suggestion
import depthai_helpers.calibration_utils as utils
```

importing using `*` can shadow names from this file. Using named import will prevent it and IMO it would be easier to understand where a function is defined